examples/raster_dataset.ipynb

```diff
@@ -161,33 +161,41 @@
     "        ),\n",
     "        Colorizer(map_name=\"gray\", min_max=(0, 255))\n",
     ")"
-   ]
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "## Store part of the world as new dataset"
-   ]
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "metadata": {},
+   "execution_count": 18,
    "outputs": [
     {
-     "data": {
-      "text/plain": [
-       "StoredDataset(dataset_id=03250e3c-8519-468a-95a6-896aa9873181, upload_id=07fb239a-65d8-4c55-b48b-7a0b8743572f)"
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "status=running, pct_complete=0.00%, time_estimate=? (± ?), info=None\n",
+      "status=completed, info={'dataset': 'b332c537-7646-44fd-8149-c8e26058844d', 'upload': '4f223ac6-681b-48ea-a0f8-431612410c94'}, time_total=00:00:00\n"
+     ]
     }
    ],
    "source": [
-    "stored_dataset = workflow.save_as_dataset(\n",
+    "task_id = workflow.save_as_dataset(\n",
     "    ge.QueryRectangle(\n",
     "        [-60.0, -10.0, 150.0, 80.0],\n",
     "        [time, time],\n",
```

Comment:
I guess you should add more than one timestamp to this example because it is possible now.

Reply:
4d71a4078feef75c8b91b16d34f0f1ef87c95c2b